full_name.rb

```diff
@@ -0,0 +1,22 @@
+class FullName
+  def initialize()
+    @name_array = []
+    print "What's your first name? "
+    @name_array.push(gets.chomp)
+    print "What's your middle name? "
+    @name_array.push(gets.chomp)
+    print "What's your last name? "
+    @name_array.push(gets.chomp)
+  end
+
+  def greet
+    full_name = ""
+    @name_array.each { |name|
+      full_name += "#{name} "
+    }
+    "Hi, #{full_name.strip}."
```

Comment:
Loooooove using `.strip` here 👏 Excellent thought to do that. Users can't be trusted 😂

Reply:
Thank you. Also for the extra space I had created by adding a space after #{name} that gets added to full_name.